### PR TITLE
add my slides

### PR DESCRIPTION
**Pull request by Xe**

### CODE REVIEW

File: .DS_Store
Comment (ryan4yin) [inline .DS_Store:1]: Should these meaningless files be removed?

Comment (ryan4yin) [reply .DS_Store:1]: I feel like we need a `.gitignore` to avoid issues like this.

Comment (marijanp) [reply .DS_Store:1]: To configure a global `.gitignore` to ignore these is a good thing to do on macos :)

Comment (kip93) [reply .DS_Store:1]: Yup, that's a good idea. Since this is not a code repo I kinda didn't set up a .gitignore. Just added a basic one to ignore .DS_Store

Comment (kip93) [reply .DS_Store:1]: So pls remove these files and we can merge this

**Pull request closed without merging**